src/driver/src/DefaultDriver.cpp

```diff
@@ -87,6 +88,67 @@ void reportMetrics(genny::metrics::Registry& metrics,
     actorSetup.report(std::move(finishTime), std::move(duration), std::move(outcome));
 }
 
+/**
+ * Filter to remove any nodes inside dedicated "ignore" values.
+ *
+ * Basically re-implementing the following python logic:
+ *  ignored = set("foo","bar")
+ *  return [u for u in unused if not any(ignore in u for ignore in ignored)]
+ */
+UnusedNodes removeIgnored(const UnusedNodes& unused, const std::vector<std::string>& ignored) {
+    UnusedNodes out{};
+    std::copy_if(
+        unused.begin(), unused.end(), std::back_inserter(out), [&](const std::string& path) {
+            return !std::any_of(ignored.begin(), ignored.end(), [&](const std::string& ignore) {
+                return path.find(ignore) != std::string::npos;
+            });
+        }
+    );
+    return out;
+}
+
+/**
+ * Log a message showing the unused yaml structures.
+ */
+void reportUnused(const NodeSource& nodeSource, const bool dryrun) {
+    auto raw = nodeSource.unused();
+
+    std::vector<std::string> ignored{".yml/Description", ".yml/Owner", ".yml/AutoRun", ".yml/Keywords"};
```

Comment:
Based on patch run add - /Clients/Default/QueryOptions/maxPoolSize to ignore list